src/v2/Apps/Fair/Routes/FairOverview.tsx

```diff
@@ -10,7 +12,33 @@ interface FairOverviewProps {
 const FairOverview: React.FC<FairOverviewProps> = ({ fair }) => {
   return (
     <>
-      <FairHeaderFragmentContainer fair={fair} />
+      <FairHeader fair={fair} />
+
+      <CSSGrid
+        gridRowGap={3}
+        gridColumnGap={3}
+        gridTemplateColumns={["repeat(1fr)", "repeat(2, 1fr)"]}
```

Comment:
Will want to use this same thing for layout in overview, instead of the bootstrap grid.

Reply:
lol I stand corrected